src/components/APIDoc/Operation.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import {OpenAPIV3} from "openapi-types";
-import {buildExample, deRef} from "../../utils/Openapi";
+import { buildExample, deRef } from '../../utils/Openapi';
```

Comment:
I guess we should add a linter to unify this - I like the space in the braces and using `'` insteasd of `"` - but lets try to have some tooling to prevent from accepting different code. 

Meanwhile I suggest to remove this change
```suggestion
import {buildExample, deRef} from "../../utils/Openapi";
```

Reply:
Ah, I think I ran `npm run build` before pushing. That might have made that unrelated change. Not sure. :thinking: